src/qibolab/instruments/qm/ports.py

```diff
@@ -1,6 +1,12 @@
 from dataclasses import dataclass, field, fields
 from typing import ClassVar, Dict, Optional, Union
 
+DIGITAL_PULSE_CALIBRATION = {"delay": 57, "buffer": 18}
+"""Calibration of digital pulses used for LO triggering.
+
+https://docs.quantum-machines.co/1.1.7/qm-qua-sdk/docs/Guides/octave/#calibrating-the-digital-pulse
+"""
```

Comment:
Shouldn't this be part of the instrument settings?

Reply:
In principle yes, but given that it is very instrument related (I don't think we are using digital pulses with any other instrument, I am not sure if they even support them) and therefore we are not going to calibrate it, at least using qibocal, I would hide it from the runcard to not confuse users. If I put it in the settings, qibocal will dump it automatically in the runcard.

As an intermediate solution, in https://github.com/qiboteam/qibolab/pull/856/commits/9e2a7b42baf516e686ea12bf7d5d35a28608e010 I moved it to port fields, so that at least it is possible to modify it when creating the platform (but in the `create` not the runcard).